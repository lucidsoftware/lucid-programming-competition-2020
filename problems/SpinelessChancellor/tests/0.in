4
P105
A461
D700
P226
3
P105 yes
A461 yes
D700 no
P226 no
P105 yes
A461 no
D700 yes
P226 unsure
P105 unsure
A461 unsure
D700 yes
P226 no