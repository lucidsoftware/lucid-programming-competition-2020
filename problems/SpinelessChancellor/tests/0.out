P105 yes
A461 no
D700 abstain
P226 no